examples/LeNet/MnistUbyte.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * \brief Mnist reader
+ * \file MnistUbyte.h
+ */
+#pragma once
+
+#include <memory>
+#include <string>
+
+/**
+ * \class MnistUbyte
+ * \brief Reader for mnist db files
+ */
+class MnistUbyte {
+  private:
+    int ReverseInt(int i);
+
+  public:
+    /**
+     * \brief Constructor of Mnist reader
+     * @param filename - path to input data
+     * @return MnistUbyte reader object
+     */
+    explicit MnistUbyte(const std::string& filename);
+    virtual ~MnistUbyte() {
+    }
+
+    /**
+     * \brief Get size
+     * @return size
+     */
+    size_t Size() const {
+        return mWidth * mHeight * 1;
+    }
+
+    void Release() {
```

Comment:
This seems to be never used? Also it's a bit weird to have the call to `delete this` inside a method instead of being done by the surrounding code.

Reply:
Good catch. It is not used. Removed it in the new commit: https://github.com/webmachinelearning/webnn-native/pull/1/commits/694cbc07a4538464ebade456cbfa143c4915e6ee